packages/mds-agency/tests/integration-tests.ts

```diff
@@ -29,19 +29,21 @@
 
 import supertest from 'supertest'
 import test from 'unit.js'
+import Sinon from 'sinon'
```

Comment:
Should not be using mocks in this test file -- this is for integration tests. We have unit tests defined in other files.